addons/binding/org.openhab.binding.enigma2/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,27 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Enigma2 Binding
+Bundle-SymbolicName: org.openhab.binding.enigma2;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.1.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .
+Import-Package: 
+ javax.jmdns;version="3.5.1",
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.types,
+ org.eclipse.smarthome.io.net.http,
+ org.eclipse.smarthome.io.transport.mdns.discovery,
+ org.openhab.binding.enigma2,
+ org.openhab.binding.enigma2.handler,
+ org.osgi.service.cm;version="1.5.0",

```

Comment:
do you need this specific version of service.cm? If not, please remove version constraint.